### ISSUE TITLE
Extra scroll bar in terminal group

### ISSUE THREAD

**Issue by yiliang114** [microsoft/vscode#167752]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.74.0 ba9399b83d048e780646896d5f81bcc08d759c6b
- OS Version: Mac 12.2.1 intel

Steps to Reproduce:

1. Clone VS Code source code from github and use VS Code to open it.
2. Use `cmd + shift + b` to start the development task,  then you can see  two task are running in `Terminal`, their names are `Core - Build` and `Ext - Build` respectively.
3. Hover mouse slides through the left icon of `Core - Build`

![bug4](https://user-images.githubusercontent.com/11473889/204870402-90d0c6e6-17b6-40e9-964b-7cd91e43f403.gif)

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! It looks like you may be using an old version of VS Code, the latest stable release is 1.73.1. Please try upgrading to the latest version and checking whether this issue remains.

Happy Coding!

**Comment by meganrogge** [comment]
I cannot reproduce, can you @Tyriar ?

**Comment by Tyriar** [comment]
@rzhao271 another custom hover scroll issue?

**Comment by meganrogge** [comment]
yes looks similar to https://github.com/microsoft/vscode/issues/166171

**Comment by rzhao271** [comment]
I'm wondering what the behaviour on Stable is for that hover?
In the screencap, it looks like there might actually be more text that needs to be displayed at the end.

I'm unable to repro the issue because when I try to run the build task on macOS, I get `yarn: command not found`. Yarn works fine when I run it outside of a task, so I'm guessing I have something misconfigured.

**Comment by meganrogge** [comment]
I was also unable to reproduce the issue

**Comment by yiliang114** [comment]
> I was also unable to reproduce the issue

@meganrogge I'm sorry, I installed VS Code and VS Code Insiders locally, I made a mistake, `VS Code  1.73.1 (Universal)` 6261075646f055b99068d3688932416f  is used in the screenshot which is the latest version that it can update at present.  

I looked at a similar problem #166171. It is a problem with 1.73.1 version. I tried it again on the insiders, but it really won't appear again, so I think it may be a fixed problem. It should be that I filled in the wrong VS Code version, which caused everyone to not reproduce. 

I'm sorry again to delay everyone's time. @Tyriar @rzhao271
